lexicons/tools/ozone/server/getConfig.json

```diff
@@ -0,0 +1,45 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.server.getConfig",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get details about ozone's server configuration.",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "appview": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "pds": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "blobDivert": {
+              "type": "ref",
+              "ref": "#serviceConfig"
+            },
+            "viewerRole": {
+              "type": "string"
+            }
```

Comment:
I wonder if we can future-proof this by making this instead: `viewer: { role }`.  That way if there is more viewer-specific info, as I expect there could be, we have a place to put it.